src/platform/forms-system/src/js/utilities/data/clone.js

```diff
@@ -6,19 +6,17 @@
  */
 export default function clone(object) {
   /* eslint-disable no-prototype-builtins */
-  switch (typeof object) {
-    case 'object':
-      if (Array.isArray(object)) {
-        return object.slice();
-      } else if (Set.prototype.isPrototypeOf(object)) {
-        return new Set(object);
-      } else if (Map.prototype.isPrototypeOf(object)) {
-        return new Map(object);
-      }
-      // Probably just ye olde object
-      return Object.assign({}, object);
-    default:
-      throw new Error(`Unknown type in clone: ${typeof object}`);
+  if (typeof object === 'object') {
```

Comment:
This `clone` function looks identical to the one in `src/platform/utilities/data/clone.js`.

Here are the files that use _this_ `clone` function:

- `src/platform/forms-system/src/js/utilities/data/cloneDeep.js`
- `src/platform/forms-system/src/js/utilities/data/index.js`
- `src/platform/forms-system/src/js/utilities/data/set.js`

Reply:
That's because it is a...um...clone. :trollface: 

Once upon a time, I wrote these data utilities. We used them in the forms system with the idea that we'd stop using lodash (never got there, obviously). Then we separated the forms system into its own repo and they copied these utilities along with them. When we moved the forms system _back_ into `vets-website`, we never refactored it to use the ones we already had.